src/components/VonayeProgress.test.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import VoyageProgress from './VoyageProgress'
+
+test('render port names', () => {
```

Comment:
There are two names here. What if the first one is correct and the second one is not. I have to go here and check what went wrong because the test name does not tells me. I'd split this up to loading and discharge.

Reply:
you mean to split it up to two different tests ?

Reply:
Yes, so whenever a test fails, you'll know exactly what went wrong instead of "this or.. this". As these are fast, it shouldn't matter if it render once or twice.